include/zephyr/arch/arm/aarch32/cortex_m/cpu.h

```diff
@@ -25,6 +25,39 @@
 /* EXC_RETURN defines */
 #define _EXC_RETURN_SPSEL_Msk (1 << 2)
 #define _EXC_RETURN_FTYPE_Msk (1 << 4)
+
+#else
+#include <zephyr/types.h>
```

Comment:
Is this include needed? I think `<stdint.h>` would be sufficient.

Reply:
it is. I'll fix it.